rpc/lib/client/ws_client.go

```diff
@@ -111,8 +111,8 @@ func (wsc *WSClient) receiveEventsRoutine() {
 				wsc.ErrorsCh <- err
 				continue
 			}
-			if response.Error != "" {
-				wsc.ErrorsCh <- errors.Errorf(response.Error)
+			if response.Error != nil && response.Error.Message != "" {
```

Comment:
Thanks. Actually, the error message should always be present. I removed the checks.